weblate/trans/models/component.py

```diff
@@ -1078,6 +1088,29 @@ def append(text: Optional[str]):
         regex = "".join(result)
         return re.compile(f"^{regex}$")
 
+    @property
+    def screenshot_filemask_re(self):
```

Comment:
[fnmatch.translate](https://docs.python.org/3/library/fnmatch.html#fnmatch.translate) does this conversion. But you don't need this at all.